lib/param_parsing/terragrunt.go

```diff
@@ -25,8 +25,8 @@ func GetVersionFromTerragrunt(params Params) (Params, error) {
 		}
 		var versionFromTerragrunt terragruntVersionConstraints
 		diagnostics = gohcl.DecodeBody(hclFile.Body, nil, &versionFromTerragrunt)
-		if diagnostics.HasErrors() {
-			return params, fmt.Errorf("could not decode body of HCL file %q", filePath)
+		if versionFromTerragrunt.TerraformVersionConstraint == "" {
+			return params, fmt.Errorf("could not find terraform_version_constraint in file %q", filePath)
```

Comment:
I might be missing something, but why would we want to return error if TG file has not explicit TF constraint? Should just ignore TG file instead in this case? 🤔

Reply:
Good idea. Implemented that